ios/run_first_test/first.conf.js

```diff
@@ -4,19 +4,19 @@ exports.config = {
 
   updateJob: false,
   specs: [
-    './tests/specs/single_test.js'
+    './run_first_test/specs/first_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'single_appium_test',
     build: 'webdriver-browserstack',
-    device: 'iPhone 7',
+    device: 'iPhone 11 Pro',
     app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
     'browserstack.debug': true
   }],
 
-  logLevel: 'verbose',
+  logLevel: 'info',
```

Comment:
Why the change from verbose to info?

Reply:
Yes, wdio v5 doesn't have loglevel verbose anymore it have to specify the level of verbose now
Options: trace | debug | info | warn | error | silent